src/pykeen/models/unimodal/complex.py

```diff
@@ -103,12 +103,14 @@ def __init__(
             # initialize with entity and relation embeddings with standard normal distribution, cf.
             # https://github.com/ttrouill/complex/blob/dc4eb93408d9a5288c986695b58488ac80b1cc17/efe/models.py#L481-L487
             entity_representations=EmbeddingSpecification(
-                embedding_dim=2 * embedding_dim,  # complex embeddings
+                embedding_dim=embedding_dim,
                 initializer=entity_initializer,
+                dtype=torch.complex64,
```

Comment:
is there a general `torch.complex`? I feel like I saw before but can't remember. I know it's not `complex` since that's a builtin

Reply:
Solution: `torch.cfloat`. See https://pytorch.org/docs/stable/complex_numbers.html#creating-complex-tensors

Reply:
Done in 64bd74e8